docs/advanced/fields.rst

```diff
@@ -151,6 +151,8 @@ function when accessing the fields:
 Now any field can be converted from sync to async, or the other way around,
 and the code would keep working.
 
+.. _`item-classes`:
```

Comment:
:lipstick: I did not know that they worked, but those quote marks are not needed here.

```suggestion
.. _item-classes:
```